package.json

```diff
@@ -60,5 +66,10 @@
   },
   "bin": {
     "uma": "./core/scripts/cli/cli_entry.sh"
+  },
+  "husky": {
+    "hooks": {
+      "pre-push": "npm run lint"
+    }
```

Comment:
Does this actually work? Wouldn't this just leave you with unstaged changes after pushing? Also, how does disabling/enabling work?

Reply:
yes, you are right. I've changed it now to `npm run lint_check` which will make the push fail if there are incorrectly linted files.